### PR TITLE
[ClockKit] Update to Xcode 9 Beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/clockkit.cs [hunk 1]
```diff
@@ -53,16 +53,20 @@ interface CLKComplicationDataSource {
 		[Export ("getTimelineEntriesForComplication:afterDate:limit:withHandler:")]
 		void GetTimelineEntriesAfterDate (CLKComplication complication, NSDate afterDate, nuint limit, Action<CLKComplicationTimelineEntry []> handler);
 	
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use WKRefreshBackgroundTask instead")]
```
Comment (VincentDondain) [inline src/clockkit.cs]: Please take a look at https://github.com/xamarin/xamarin-macios/wiki/BINDINGS#availability-attributes-messages as a refresher (:

Your message should be `Use 'WKRefreshBackgroundTask' instead.`

File: src/clockkit.cs [hunk 2]
```diff
@@ -53,16 +53,20 @@ interface CLKComplicationDataSource {
...
 		[Export ("getNextRequestedUpdateDateWithHandler:")]
 		void GetNextRequestedUpdateDate (Action<NSDate> handler);
 
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use `WKRefreshBackgroundTask` instead")]
```
Comment (VincentDondain) [inline src/clockkit.cs]: https://github.com/xamarin/xamarin-macios/wiki/BINDINGS#rule-2 please use apostrophes not grave accent (and add a dot at the end) (:

File: src/clockkit.cs [hunk 3]
```diff
@@ -53,16 +53,20 @@ interface CLKComplicationDataSource {
...
 		[Export ("requestedUpdateDidBegin")]
 		void RequestedUpdateDidBegin ();
 
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use `WKRefreshBackgroundTask` instead")]
```
Comment (VincentDondain) [inline src/clockkit.cs]: Same

File: src/clockkit.cs [hunk 4]
```diff
@@ -53,16 +53,20 @@ interface CLKComplicationDataSource {
...
 		[Export ("requestedUpdateBudgetExhausted")]
 		void RequestedUpdateBudgetExhausted ();
 
 		// this was @required in watchOS 2.x but is now deprecated and downgraded to @optional in watchOS 3 (betas)
+		[Deprecated (PlatformName.WatchOS, 3, 0, message: "Use `GetLocalizableSampleTemplate` instead")]
```
Comment (VincentDondain) [inline src/clockkit.cs]: Same

**Comment by dalexsoto** [comment]
@VincentDondain Done! And thanks for the refresher!

**Pull request merged**